app/helpers/form_helper.rb

```diff
@@ -0,0 +1,6 @@
+module FormHelper
+  def list_of_organisations
```

Comment:
I predict we'll have to do more complicated logic here in the future. Defaulting the `receiver` as BEIS when creating a fund and the `provider` when creating a project level activity.